packages/common/src/controllers/CheckoutController.ts

```diff
@@ -174,18 +191,26 @@ export default class CheckoutController {
       paymentData,
     });
 
-    if (response.errors.length > 0) throw new Error(response.errors[0]);
+    if (response.errors.length > 0) {
+      throw new Error(response.errors[0]);
+    }
 
     return response.responseData;
   };
 
-  paypalPayment = async (
-    successUrl: string,
-    waitingUrl: string,
-    cancelUrl: string,
-    errorUrl: string,
-    couponCode: string = '',
-  ): Promise<PaymentWithPayPalResponse> => {
+  paypalPayment = async ({
+    successUrl,
+    waitingUrl,
+    cancelUrl,
+    errorUrl,
+    couponCode = '',
+  }: {
+    successUrl: string;
+    waitingUrl: string;
+    cancelUrl: string;
+    errorUrl: string;
+    couponCode: string;
+  }): Promise<string> => {
```

Comment:
If we remove the response type, can we at least create an object type for the response here (i.e. `{ redirectUrl: string}`? Returning a raw string is a little hard to interpret for callers of this method.